test/helpers/cilium.go

```diff
@@ -333,6 +349,96 @@ func (c *Cilium) ReportFailed(commands ...string) {
 		fmt.Fprint(wr, res.Output())
 	}
 	fmt.Fprint(wr, "StackTrace Ends\n")
+	c.ReportFailedFull()
+}
+
+//ReportFailedFull runs a variety of commands and write the results to
+//testResultsPath
+func (c *Cilium) ReportFailedFull() {
+	reportCommands := map[string]string{
```

Comment:
This is duplicated here: https://github.com/cilium/cilium/pull/2038/files#diff-2304891712e5d0140828499fc3a19043R447 . Please reduce the amount of duplicate code if possible and refactor accordingly.